python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -242,19 +276,34 @@ def post_conv_psf_ft2(psf, kernel, sig1=1., sig2=1.):
                 kernel = np.pad(kernel, (diff, diff), mode='constant')
             psf_ft = fft2(psf)
             kft = fft2(kernel)
-            out = psf_ft * np.sqrt((sig1**2 + sig2**2) / (sig1**2 + sig2**2 * kft**2))
+            out = psf_ft * np.sqrt((var1 + var2) / (var1 + var2 * kft**2))
             return out
 
-        def post_conv_psf(psf, kernel, sig1=1., sig2=1.):
-            kft = post_conv_psf_ft2(psf, kernel, sig1, sig2)
+        def post_conv_psf(psf, kernel, var1=1., var2=1.):
+            kft = post_conv_psf_ft2(psf, kernel, var1, var2)
             out = ifft2(kft)
             return out
 
-        pcf = post_conv_psf(psf=psf, kernel=kappa, sig1=sig2, sig2=sig1)
+        pcf = post_conv_psf(psf=psf, kernel=kappa, var1=var1, var2=var2)
         pcf = pcf.real / pcf.real.sum()
         return pcf
 
     @staticmethod
+    def _fixOddKernel(kernel):
```

Comment:
Not sure this should be a static method: it looks more like a stand-alone, top-level function.